onesignalclient/notification.py

```diff
@@ -43,6 +46,31 @@ def include_player_ids(self, value):
 
         self._include_player_ids = value
 
+    @property
+    def include_segments(self):
+        return self._include_segments
+
+    # notification.include_segments = [Notification.SEGMENT_ALL]    
```

Comment:
Is this comment really necessary? If so, we can add it to the function following the PEP indication for doc comments. https://www.python.org/dev/peps/pep-0257/